src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("/fonts/Avenir-Heavy.ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+:root {
+  --text-color: rgb(61, 78, 97);
+  --bg-gradi: linear-gradient(180deg, rgb(255, 255, 255), rgb(246, 246, 247));
+}
+
+body {
+  margin: 0;
+  font-weight: 300;
+  font-family: "Avenir Book", Helvetica, Arial, sans-serif;
+}
+
+::placeholder {
+  color: var(--text-color);
+}
+
+.search-bar {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  width: 100%;
+}
+
+.search-bar__big {
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  width: 100%;
```

Comment:
it doesn't work
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/86923089/9bfa6677-b8b3-4db2-bede-8a7328a0c1a1)

Reply:
Still not fixed